test/integration/targets/nxos_file_copy/tests/cli/sanity.yaml

```diff
@@ -57,6 +58,37 @@
     nxos_command: *remove_file
     register: result
 
+# Uncomment the following tasks with proper filename, scp server information
```

Comment:
Is there any specific reason that the test is uncommented?

Reply:
I believe you meant, why this test is ```commented``` out.
For file copy from the switch, we need to provide the remote_scp_server, remote_scp_username, remote_scp_password, remote_file_name etc. I tested this using my login credentials and the file_servers in our labs. We cannot put them in the test file as they are specific to our lab. So we cannot ```uncomment``` these tests until the real information of these parameters is given. I only put some example information there for now and commented it out.